src/metrics/time-to-merge.ts

```diff
@@ -0,0 +1,48 @@
+import { Duration, Interval } from 'luxon';
+import { fetchMergedPullRequestNumbers } from '../utils/graphql-queries';
+import { loadPullRequest } from '../models/pull-request';
+import debug, { Debugger } from '../utils/debug';
+import durationToHuman from '../utils/duration-to-human';
+import { Metric, MetricData, percentiles } from '../metric';
+
+export default class TimeToMergeMetric implements Metric {
+  debug: Debugger;
+  data: MetricData[];
+  name = 'Pull Request Time-To-Merge';
+  constructor(public interval: Interval) {
+    this.debug = debug.extend('metrics:time-to-merge');
+    this.data = [];
+  }
+
+  get summary(): string {
+    let values = this.data.map((row) => row.value);
+    if (values.length === 0) {
+      return `No data`;
```

Comment:
```suggestion
      return 'No data';
```

Reply:
👍  we should probably use prettier for linting so this is automatic